packages/jsapi-components/src/useGetItemIndexByValue.ts

```diff
@@ -38,7 +38,7 @@ export function useGetItemIndexByValue<TValue>({
       return index === -1 ? null : index;
     } catch (err) {
       log.debug('Error seeking row', { column, value, columnValueType });
-      return null;
+      throw err;
```

Comment:
I wouldn't bother logging it either - consumer either catches it and logs it or does something with it in which case the log would be extra spam. If it's uncaught would be logged as an uncaught promise anyway.